src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -90,6 +90,10 @@ internal partial class Binder
 
             // Obsolete diagnostics for method group are reported as part of creating the method group conversion.
             ReportDiagnosticsIfObsolete(diagnostics, conversion, syntax, hasBaseReceiver: false);
+            if (conversion.Method is not null)
```

Comment:
>if (conversion.Method is not null) [](http://example.com/codeflow?start=12&length=34)

There are many other places where we create Conversion objects like this, but do not pass them to this method. I am planning to remedy that in 17.1.  So, I think it is fine limit the fix to this code path.